### PR TITLE
Prettify Users page

### LINKED ISSUE #4239: Prettify the Users page
Prettify the search bar

Users

### PR DESCRIPTION
**Pull request by hishamco**
Fix #4239

Users

### CODE REVIEW

**Comment by hishamco** [comment]
I file two related issues that we need finish before 1.0.0 release if it's possible

#4242 #4243

**Comment by hishamco** [comment]
:shipit:

File: src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -1,58 +1,47 @@
...
-    <form asp-action="Index" method="get" class="form-inline">
-        <input id="search-box" asp-for="Options.Search" class="form-control mr-sm-2 my-2 my-lg-0" type="text" placeholder="@T["Search"]" autofocus="autofocus">
-        <button type="submit" class="btn btn-secondary">@T["Search"]</button>
-    </form>
-</nav>
+        </div>
+    </fieldset>
+</form>
 
-@using (Html.BeginForm())
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/Index.cshtml]: You are removing the form. does it still work?

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/Index.cshtml]: Yep, coz if you look to the search bar it surroundings with a form, may be the deleted form was added for a reason in the past and no one removed

**Comment by Skrypt** [comment]
```
@model UsersIndexViewModel

<div class="row">
    <div class="form-group col mb-n1">
        <h1>@RenderTitleSegments(T["Users"])</h1>
    </div>
    <div class="form-group col mb-n1">
        <div class="dropdown float-right order-md-1">
            <a asp-route-action="Create" class="btn btn-primary float-right" role="button">@T["Add User"]</a>
        </div>
    </div>
</div>

<form asp-action="Index" method="get">
    <div class="card mb-3">
        <div class="card-body bg-light p-3">
            <div class="input-group input-group-sm w-lg-50">
                <div class="input-group-prepend">
                    <span class="input-group-text bg-secondary text-light">@T["Filter"]</span>
                </div>
                <input id="search-box" asp-for="Options.Search" class="form-control" placeholder="@T["Search"]" autofocus="autofocus" />
                <div class="input-group-append">
                    <button type="submit" class="btn btn-sm btn-secondary">@T["Search"]</button>
                </div>
            </div>
        </div>
    </div>
</form>

@if (Model.Users.Count > 0)
{
    <ul class="list-group">
        @foreach (var entry in Model.Users)
        {
            <li class="list-group-item">
                @await DisplayAsync(entry.Shape)
            </li>
        }
    </ul>
}
else
{
    <div class="alert alert-info" role="alert">
        @T["<strong>Nothing here!</strong> Your search returned no result."]
    </div>
}

@await DisplayAsync(Model.Pager)
```

**Comment by hishamco** [comment]
Thanks Jasmin

**Comment by sebastienros** [comment]
Thanks Jasmin. @hishamco would you mind updating your recent changes and open PRs to follow this pattern? Thank you.

**Comment by hishamco** [comment]
Sure

**Comment by hishamco** [comment]
It would be nice if we make the alert dismissable

**Pull request merged**